Source/ProjectFirma.Web/ProjectFirma.Web.csproj

```diff
@@ -707,8 +710,13 @@
     <Compile Include="Models\Generated\WashingtonLegislativeDistrict.Binding.cs" />
     <Compile Include="Models\Generated\WashingtonLegislativeDistrict.DatabaseContextExtensions.cs" />
     <Compile Include="Models\Generated\WashingtonLegislativeDistrictPrimaryKey.Binding.cs" />
+<<<<<<< HEAD

```

Comment:
Looks like the conflict here wasn't completely resolved

Reply:
done